core/constants-dev.yaml

```diff
@@ -2,7 +2,7 @@
 k: 2
 
 # Genesis constants
-genesisN: 12
+genesisN: 50000
```

Comment:
logs will be large and unreadable (at least during first epochs) because we print ssc global state on each slot